.template/variants/web/.gitignore.rb

```diff
@@ -26,5 +26,9 @@
 
     # Ignore Gemfile.lock file in engines.
     /engines/*/Gemfile.lock
+
+    # Ignore asset builds
+    /app/assets/builds/*
+    !/app/assets/builds/.keep
```

Comment:
@andyduong1920 , it's already there at the beginning of this file, do we need to add it again? 🤔 

https://github.com/nimblehq/rails-templates/blob/1a3bbf565ea1702e91f43c14e14cf44b3fb80475/.template/variants/web/.gitignore.rb#L9-L11

Reply:
my bad @tyrro removed it 🤞